src/DynamoInstall/Release.xsl

```diff
@@ -40,6 +34,14 @@
     <!--Exclude binariestosign.txt-->
     <xsl:key name="binariestosign-search" match="wix:Component[contains(wix:File/@Source, 'binariestosign.txt')]" use="@Id"/>
     <xsl:template match="wix:Component[key('binariestosign-search', @Id)]" />
+	
+	<!--Exclude RevitAddinUtility.dll-->
```

Comment:
Can we also figure out the build step which is copying Revit specific files to bin folder. Just want to make sure that Revit specific files are not getting copied while building DynamoDS\Dynamo.